src/Credentials.js

```diff
@@ -119,12 +285,14 @@ class Credentials {
  *    ...
  *  })
  *
- *  @param    {Object}              unsignedClaim       an object that is an unsigned claim which you want the user to attest
- *  @param    {String}             sub                  the DID of the identity you want to sign the attestation
- *  @return   {Promise<Object, Error>}                  a promise which resolves with a signed JSON Web Token or rejects with an error
+ *  @param    {Object}      unsignedClaim     an object that is an unsigned claim which you want the user to attest
+ *  @param    {String}      aud               the DID of the identity you want to sign the attestation
+ *  @param    {String}      sub               the DID which the unsigned claim is about
+ *  @param    {String}      callbackUrl       the url which you want to receive the response of this request
+ *  @return   {Promise<Object, Error>}        a promise which resolves with a signed JSON Web Token or rejects with an error
  */
-  createVerificationRequest(unsignedClaim, sub) {
-    return createJWT(this.settings, {unsignedClaim, sub, type: 'verReq'})
+  createVerificationRequest(unsignedClaim, sub, callbackUrl, aud) {
```

Comment:
the name is very hard to differentiate from the selective disclosure request, it sounds like a function to request verifications, not one for signing claims. Not sure when we started using verifications every where along with already many names including claims and attestations (I find the spec confusing for this request in that respect as well) 

Maybe we could make our naming more clear and uniform if we used similar naming for all request related funds and then also responses related funcs. I.e requestDisclosure, then this could be requestSignClaim or requestSignVerification

Reply:
What if we went with `requestSignature(unsignedClaim, ...)`?  I like Zach's point about keeping `request`s named similarly.  Also the argument being named `unsignedClaim` I think that makes it fairly clear that the function is byo claim

Reply:
Rename to `createSignVerificationRequest()`